src/elements/LicenseIconography/LicenseIconography.vue

```diff
@@ -26,7 +32,7 @@
     faCreativeCommonsShare
   } from '@fortawesome/free-brands-svg-icons'
   import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
-
+  import { Popup } from '@/patterns/Popup/Popup'
```

Comment:
Leave a line between imports from third parties and from within the project. Also leave a line between imports and the rest of the JavaScript.

Reply:
```suggestion
  import Popup from '@/patterns/Popup/Popup'
```